jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/ExamplesTableUtil.java

```diff
@@ -142,6 +144,28 @@ public static String getValue(ExamplesTable table, String keyColumn, String key,
             .get(valueColumn);
     }
 
+    /**
+     * Checks whether examplesTable has all mandatory columns
+     *
+     * @param examplesTable     table to be checked
+     * @param expectedColumns   list of mandatory columns
+     *
+     * @throws org.assertj.core.api.SoftAssertionError   for every column missing in table
+     * @throws java.lang.IllegalArgumentException        when expectedColumns is missing
+     */
+    public static void assertMandatoryColumns(ExamplesTable examplesTable, String ... expectedColumns) {
+        if (expectedColumns.length < 1){
+            throw new IllegalArgumentException("expectedColumns must not be empty");
+        }
```

Comment:
`Assert.notEmpty(expectedColumns, "expectedColumns must not be empty");`

Reply:
My imagining of Dan's reaction to every new comment:
![](https://media3.giphy.com/media/l2JhtKtDWYNKdRpoA/giphy.gif)

Reply:
I think this reaction is more accurate:
![image](https://user-images.githubusercontent.com/50624143/69132304-1de29100-0ab4-11ea-888a-2f7a3876857f.png)
But going to fix it, looks way better Amy's way.

Reply:
done :)